src/stirling/source_connectors/socket_tracer/openssl_trace_bpf_test.cc

```diff
@@ -77,7 +83,7 @@ struct TraceRecords {
   std::vector<std::string> remote_address;
 };
 
-template <typename TServerContainer, bool TForceFptrs>
+template <typename TServerContainer, bool TUseNewImpl>
```

Comment:
We should just give the `bool` parameter a normal name. I'm assuming the `T` prefix means type in other contexts, but this is not a type parameter.

Can we use a name like `UseNestedSyscallFD`? because even if imperfect, that name describes the feature better than "NewImpl" which may become confusing in the future.

Reply:
> I'm assuming the T prefix means type in other contexts, but this is not a type parameter.

I wasn't sure what our convention was, but since I see `TEnableClientSideTracing` frequently I followed suit.

```
ddelnano@vigenere:~/code/pixie (ddelnano/add-feature-flag-to-opt-into-new-tls-tracing-method) $ git grep TEnableClientSideTracing
src/stirling/source_connectors/socket_tracer/testing/socket_trace_bpf_test_fixture.h:template <bool TEnableClientSideTracing = false>
src/stirling/source_connectors/socket_tracer/testing/socket_trace_bpf_test_fixture.h:    FLAGS_treat_loopback_as_in_cluster = !TEnableClientSideTracing;
src/stirling/source_connectors/socket_tracer/testing/socket_trace_bpf_test_fixture.h:    if (TEnableClientSideTracing) {
```

From searching the rest of the codebase, it doesn't appear we use that pattern elsewhere. So we should separately rename `TEnableClientSideTracing` to `EnableClientSideTracing`, which I did in https://github.com/pixie-io/pixie/pull/1125.

I initially used something similar to `UseNestedSyscallFD`, but opted for `TUseNewImpl` since this will eventually be removed once we fully roll out the new tracing method and clean up the temporary code. However, I agree that referring to what the implementation does is a better identifier so I'll update this.

Reply:
This was updated in a later revision.